cmd/kpod/history.go

```diff
@@ -0,0 +1,267 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+	"text/template"
+	"time"
+
+	"os"
+
+	"strconv"
+
+	"github.com/Sirupsen/logrus"
+	is "github.com/containers/image/storage"
+	"github.com/containers/storage"
+	units "github.com/docker/go-units"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+// historyOutputParams stores info about each layer
+type historyOutputParams struct {
+	ID        string
+	Created   string
+	CreatedBy string
+	Size      string
+	Comment   string
+}
+
+// historyOptions stores cli flag values
+type historyOptions struct {
+	image   string
+	human   bool
+	noTrunc bool
+	quiet   bool
+	format  string
+}
+
+var (
+	historyFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "human, H",
+			Usage: "Display sizes and dates in human readable format",
+		},
+		cli.BoolFlag{
+			Name:  "no-trunc",
+			Usage: "Do not truncate the output",
+		},
+		cli.BoolFlag{
+			Name:  "quiet, q",
+			Usage: "Display the numeric IDs only",
+		},
+		cli.StringFlag{
+			Name:  "format",
+			Usage: "Pretty-print history of the image using a Go template",
+		},
+	}
+
+	historyDescription = "Displays the history of an image. The information can be printed out in an easy to read, " +
+		"or user sepcified format, and can be truncated."
+	historyCommand = cli.Command{
+		Name:        "history",
+		Usage:       "Show history of a specified image",
+		Description: historyDescription,
+		Flags:       historyFlags,
+		Action:      historyCmd,
+		ArgsUsage:   "",
+	}
+)
+
+func historyCmd(c *cli.Context) error {
+	store, err := getStore(c)
+	if err != nil {
+		return err
+	}
+
+	human := true
+	if c.IsSet("human") {
+		human = c.Bool("human")
+	}
+	noTruncate := false
+	if c.IsSet("no-trunc") {
+		noTruncate = c.Bool("no-trunc")
+	}
+	quiet := false
+	if c.IsSet("quiet") {
+		quiet = c.Bool("quiet")
+	}
+	format := ""
+	hasTemplate := false
```

Comment:
No need for hasTemplate, either check 
if c.IsSet("format")  

below, or check if format == ""